example.html

```diff
@@ -54,12 +54,18 @@
       padding: 0 5px;
       margin: 20px 0;
     }
+	.video-js 
```

Comment:
Looks like you may be using tab-based indentation. This project uses 2-space indent